gitpod-io/website
# fix(jetbrains): link to indexing per IDE

## Description
Link to external docs should be per IDE product, at the moment it always send users to the GoLand specific page.

## Related Issue(s)

Fixes #2232

## How to test
1. Visit https://deploy-preview-2234--gitpod-kumquat.netlify.app/docs/ides-and-editors
2. Check each JB IDE product page and verify that the link to `general recommendations` links to the right IDE

## Release Notes

```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2234"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2234"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
